hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketSetPropertyRequest.java

```diff
@@ -322,6 +330,13 @@ public boolean checkQuotaNamespaceValid(OmVolumeArgs omVolumeArgs,
     if (quotaInNamespace < OzoneConsts.QUOTA_RESET || quotaInNamespace == 0) {
       return false;
     }
+    
+    if (quotaInNamespace != OzoneConsts.QUOTA_RESET
+        && quotaInNamespace < dbBucketInfo.getUsedNamespace()) {
+      throw new OMException("Cannot update bucket quota. NamespaceQuota " +
+          "requested is less than used namespaceQuota.",
+          OMException.ResultCodes.QUOTA_ERROR);
+    }
```

Comment:
Should we also add a volume-level check? What if the volume level namespace is lower than all the combined bucket namespaces? Looks like `checkQuotaBytesValid` does this.